conf/app.routes

```diff
@@ -34,10 +34,8 @@ GET         /account/signed-out
 
 GET         /unauthorised                                                                                      controllers.UnauthorisedController.onPageLoad()
 
-GET         /:srn/what-you-will-need/:fbNumber                                                                 controllers.WhatYouWillNeedController.onPageLoad(srn: Srn, fbNumber: String, taxYear: String = "", version: String = "")
-GET         /:srn/what-you-will-need/:taxYear/:version                                                         controllers.WhatYouWillNeedController.onPageLoad(srn: Srn, fbNumber: String = "", taxYear: String, version: String)
-GET         /:srn/what-you-will-need                                                                           controllers.WhatYouWillNeedController.onPageLoad(srn: Srn, fbNumber: String = "", taxYear: String = "2022-04-06", version: String = "001")
-GET         /:srn/what-you-will-need-next-page                                                                 controllers.WhatYouWillNeedController.onSubmit(srn: Srn, taxYear: String, fbNumber: String,  version: String)
+GET         /:srn/what-you-will-need                                                                           controllers.WhatYouWillNeedController.onPageLoad(srn: Srn)
```

Comment:
Is any of these route changes require changes in acceptance tests?

Reply:
Acceptance tests passed locally